app/assets/javascripts/map.js

```diff
@@ -1,9 +1,12 @@
-// function initialize() {
-//   var mapOptions = {
-//     center: centerPos,
-//     zoom: 8
-//   };
-//   var map = new google.maps.Map(document.getElementById('map-canvas'),
-//       mapOptions);
-// }
-// google.maps.event.addDomListener(window, 'load', initialize);
+$(function() {
+  function initialize() {
+    var mapOptions = {
+      center: { lat: center_location.latitude, lng: center_location.longitude },
```

Comment:
Identifier 'center_location' is not in camel case.<br>'center_location' is not defined.